shell/platform/linux/sequential_id_generator.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This class makes this very hard to understand. It's copied from another project, it's in C++ style which doesn't fit in well with the other code. A number of methods aren't being used. It's not used by any other code. I think this could be simplified and merged into FlTouchManager, though that's not a blocker and could be done at a later stage.